src/main/java/seedu/weme/model/UserPrefs.java

```diff
@@ -80,6 +82,15 @@ public void setTemplateImagePath(Path templateImagePath) {
         this.templateImagePath = templateImagePath;
     }
 
+    public Path getStatsDataFilePath() {
+        return statsDataFilePath;
+    }
+
+    public void setStatsDataFilePath(Path likeDataFilePath) {
+        requireNonNull(likeDataFilePath);
+        this.statsDataFilePath = likeDataFilePath;
```

Comment:
im assuming this will be changed when you add other stats in?

Reply:
Ugh it's just naming stuff. Fixed.